src/core/server/plugins/plugin_context.ts

```diff
@@ -155,7 +157,8 @@ export function createPluginSetupContext<TPlugin, TPluginDependencies>(
         null,
         plugin.opaqueId
       ),
-      createRouter: () => deps.http.createRouter('', plugin.opaqueId),
+      createRouter: () => router,
+      resources: deps.httpResources.createRegistrar(router),
```

Comment:
For end-users, it looks like `httpResources` is a sub-service of `http` service. There is a discussion why it's not https://github.com/elastic/kibana/pull/61797/files#r403147133